src/main/scala/io/pravega/connectors/spark/PravegaSourceProvider.scala

```diff
@@ -96,6 +96,7 @@ object PravegaSourceProvider extends Logging {
   private[spark] val DEFAULT_BATCH_TRANSACTION_TIMEOUT_MS: Long = 2 * 60 * 1000 // 2 minutes (maximum allowed by default server)
   private[spark] val DEFAULT_TRANSACTION_STATUS_POLL_INTERVAL_MS: Long = 50
 
+  private[spark] val MAX_OFFSET_PER_TRIGGER = "maxoffsetspertrigger"

```

Comment:
It would be better if we align with Pravega convention for the key naming, which is separating with `_`

Reply:
This should be same as one expected from spark so that it will pick its value from spark side?
We will be passed its value from longevity.

Reply:
> This should be same as one expected from spark so that it will pick its value from spark side?

No, this is just keeping same as Kafka which is not needed.

Reply:
done